photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriver.scala

```diff
@@ -637,21 +635,34 @@ object GameTrainingDriver extends GameDriver {
         // TODO: Match on this to make it clearer
         val evaluator = models.head._2.get.head._1
         val baseConfig = models.head._3
-        val evaluationFunction = new GameEstimatorEvaluationFunction(estimator, baseConfig, trainingData, testData)
 
         val range = getOrDefault(hyperParameterTuningRange)
-        val ranges = List.fill(evaluationFunction.numParams)(DoubleRange(log(range.start), log(range.end)))
+        val rangeWrapped = DoubleRange(log(range.start), log(range.end))
+
+        val evaluationFunction = new GameEstimatorEvaluationFunction(estimator, baseConfig, trainingData, testData, range)
```

Comment:
Shouldn't you call this with the rangeWrapped? As the candidate is scaled by this range before evaluation. So technically, we should scale [0,1] -> [-3, 3] -> then take exp. Otherwise we are directly scaling [0,1] -> [0.001, 1000]

Reply:
Yes, it's a typo. It should be rangeWrapped in line 642